book/coloring_and_theming.md

```diff
@@ -34,21 +28,27 @@ Here are the current options for `$env.config.table.mode`:
 - `none`
 - `other`
 
-### Color Symbologies
+## Color Configuration
+
+The color configuration is defined in `$env.config.color_config`. The current configuration can be printed with:
+
+```nu
+$env.config.color_config | sort
```

Comment:
sort is not required

Reply:
It's not required, but useful. Without it the output is not sorted.